### ISSUE TITLE
A bit stutters while scrolling pages

### ISSUE THREAD

**Issue by eirmd** [raulhaag/MiMangaNu#552]
App version: 1.86
Android: 8.1 (xiaomi mi max)

Issue/Request: a bit stutters while scrolling pages (not smooth)

The scroll less smooth. I try to change max texture size and scrolling speed but still not solve the issue.

**Comment by raulhaag** [comment]
Sorry for be late, about this, is the continuous reader? the image loading? can be more specific from where, doing what, etc.

**Comment by eirmd** [comment]
yes, continous reader. image loading fine. while i am scrolling, its not smooth.. stutter a bit. it happen to all comic. i dont know if happen because of my device, but there's no problem for other manga reader.

**Comment by eirmd** [comment]
i set read direction from up down. when continuous reader turn off, scrolling smooth. it happens when continuous reader turn on..

**Comment by Nesaijn** [comment]
I also experience the same issue. I use v1.112 of Mi Manga Nu. 
It happens with Mangas where the page is too big to just swipe through, like in korean Mangas. Two examples where I experienced this issue are "I am the Sorcerer King" and "Peerless Dad". When you move through page by dragging, it stutters a lot. The moving from one page to the other is smooth tough.
